src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1021,6 +1021,8 @@
     <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
 
     <Field Name="ParameterSymbol" Type="ParameterSymbol"/>
+    <!-- If this is a compiler generated parameter reference created for a range variable access in query expression, then this contains the range variable -->
+    <Field Name="AssociatedRangeVariable" Type="RangeVariableSymbol" Null="allow"/>
```

Comment:
><Field Name="AssociatedRangeVariable" Type="RangeVariableSymbol" Null="allow"/> [](start = 4, length = 79)

I think we don't actually need this property because, often, the parameter itself is not associated with a range variable, but a property accessed of off the parameter is associated instead.